pandas/tests/test_tseries.py

```diff
@@ -316,6 +316,29 @@ def test_maybe_booleans_to_slice():
     assert(result == slice(0, 0))
 
 
+def test_datetimelike_tostring_astype():
```

Comment:
these needs need to go in ``tseries/tests/test_base.py`` in the ``TestDatetimeIndexOps``, right above or below the repr ones.